I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 264, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\PerezRomero\Downloads\renpy-7.1.0-sdk\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "C:\Users\PerezRomero\Downloads\renpy-7.1.0-sdk\renpy\main.py", line 513, in main
    run(restart)
  File "C:\Users\PerezRomero\Downloads\renpy-7.1.0-sdk\renpy\main.py", line 139, in run
    renpy.execution.run_context(True)
  File "C:\Users\PerezRomero\Downloads\renpy-7.1.0-sdk\renpy\execution.py", line 876, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 264, in script
    python:
  File "C:\Users\PerezRomero\Downloads\renpy-7.1.0-sdk\renpy\script.py", line 873, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-8-6.2.9200
Ren'Py 7.1.0.882
Prueba Prueba Literature Club 1.5
Fri Oct 12 23:54:56 2018
